packages/editor/src/components/post-title/index.js

```diff
@@ -63,6 +69,14 @@ export default function PostTitle() {
 		};
 	}, [] );
 
+	useImperativeHandle( forwardedRef, () => ( {
+		focus: () => {
+			if ( isCleanNewPost ) {
```

Comment:
I think this check should move outside of "post-title" component. Because it's not clear why "calling" ref.focus should abort if the post is new. So we can move this logic instead to https://github.com/WordPress/gutenberg/pull/40195/files#diff-175b6822ebac715bc984ae982f1d9230b5d1e441c98138d8a1fc7ccf275c19bcR194 (or remove it entirely)

Reply:
@youknowriad I thought it would be better to keep it there since it was already selected. Anyway, moved it to edit-post.